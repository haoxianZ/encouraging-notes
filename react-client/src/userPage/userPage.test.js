import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import UserPage from './userPage'

describe(`AddNote component`, () => {

  it('renders the complete form', () => {
    const wrapper = shallow(<UserPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})
