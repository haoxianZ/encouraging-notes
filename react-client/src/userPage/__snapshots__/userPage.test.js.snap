// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddNote component renders the complete form 1`] = `
<section
  className="userPage"
>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/"
  >
    <h2>
      Encouragement Bank
    </h2>
  </Link>
  <nav>
    <h3>
      Notes that I owned
    </h3>
  </nav>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="newNote"
    >
      Deposit an Encouragement:  
    </label>
    <textarea
      id="newNote"
      name="newNote"
      required={true}
      type="textarea"
    />
    <button
      className="submitBtn"
      type="submit"
    >
      Exchange Note
    </button>
  </form>
</section>
`;
